source/Ship.cpp

```diff
@@ -2662,7 +2662,8 @@ int Ship::RequiredCrew() const
 
 int Ship::CrewValue() const
 {
-	return max(Crew(), RequiredCrew()) + attributes.Get("crew equivalent");
+	int crewEquivalent = attributes.Get("crew equivalent");
+	return crewEquivalent ? crewEquivalent : max(Crew(), RequiredCrew());
```

Comment:
If we're to add a new attribute on top of this one that allows explicit zeros, then I think we should something like this instead.
```suggestion
	int crewEquivalent = attributes.Get("crew equivalent")
	if(attributes.get("crew equivalent overrides"))
		return crewEquivalent ? crewEquivalent : max(Crew(), RequiredCrew());
	return max(Crew(), RequiredCrew()) + crewEquivalent;
```

This keeps the existing behavior of this attribute, but if you want to instead have the crew equivalent replace the crew amount instead of add to it, you add the "crew equivalent overrides" attribute. (Open to suggestions for different names.) This also allows explicit zeros by having the overrides attribute but no crew equivalent attribute.